### PR TITLE
FHIR IG Allergy Intolerance Resource, Profile, Mapping and Example Resource

### PR DESCRIPTION
**Pull request by sherrif10**
https://issues.openmrs.org/browse/FM2-505
cc  @ibacher  @mozzy11  @pmanko  
allergy
allergy1
allergyy

### CODE REVIEW

File: input/fsh/omrs-allergyIntolerance.fsh [hunk 1]
```diff
@@ -0,0 +1,67 @@
+Profile:  OpenMRSAllergyIntolerance
+Parent:   AllergyIntolerance
+Id:       omrs-AllergyIntolerance
+Title:    "OpenMRS AllergyIntolerance"
+Description: "A FHIR Allergy as understood by OpenMRS"
+* identifier 0..1
+* code 0..1
+* category 0..*
```
Comment (pmanko) [inline input/fsh/omrs-allergyIntolerance.fsh:8]: I don't see `category`, `encounter`, `onset`, `asserter`, `lastOccurance` mapped in the translator.

**Review by pmanko — CHANGES_REQUESTED** [review]
Double-check the fields to make sure we support all of the ones listed in this IG

File: input/fsh/omrs-allergyIntolerance.fsh (hunk 1 above)
Comment (sherrif10) [reply input/fsh/omrs-allergyIntolerance.fsh:8]: oh how i missed it. thanks, Let me add them.

Comment (sherrif10) [reply input/fsh/omrs-allergyIntolerance.fsh]: Resolved them. Thanks @pmanko .

Comment (abertnamanya) [reply input/fsh/omrs-allergyIntolerance.fsh]: > Let me remove them.

Hi @sherrif10  don't remove the attributes which are not supported by fhir2 just give them the cardinality of 0..0

File: input/fsh/omrs-allergyIntolerance.fsh [hunk 2]
```diff
@@ -5,19 +5,16 @@ Title:    "OpenMRS AllergyIntolerance"
...
-* category 0..*
 * reaction 0..1
 * type  0..1
 * clinicalStatus 0..1
 * verificationStatus 0..1
+* patient 1..1
+* recorder 0..1
+* type 0..1
 * criticality 0..1
-* encounter 0..1
```
Comment (abertnamanya) [inline input/fsh/omrs-allergyIntolerance.fsh]: Hi, Sharif don't completely remove the attributes which are not supported , provide 0..0 cardinality

**Pull request merged**